ArchiSteamFarm/Utilities.cs

```diff
@@ -183,6 +196,24 @@ public static class Utilities {
 			return (text.Length % 2 == 0) && text.All(Uri.IsHexDigit);
 		}
 
+		[ItemNotNull]
+		[NotNull]
+		[PublicAPI]
+		public static List<IElement> SelectElementNodes([NotNull] this IElement element, string xpath) => element.SelectNodes(xpath).Cast<IElement>().ToList();
+
+		[ItemNotNull]
+		[NotNull]
+		[PublicAPI]
+		public static List<IElement> SelectNodes([NotNull] this IDocument document, string xpath) => document.Body.SelectNodes(xpath).Cast<IElement>().ToList();
```

Comment:
I wonder if this function and the one above could be merged to `List<IElement> SelectNodes(this INode node, string xpath)` as they both derive from `INode`

Reply:
Good eye, @Vital7 ?

Reply:
I think it shouldn't would work, `SelectNodes` works with `IElement`, not with `INode` generally, and we can't cast `IDocument` to `IElement`.

Reply:
Giving a second look at it, yeah, I don't think we can improve this part further.